types/stylelint/index.d.ts

```diff
@@ -312,10 +326,14 @@ declare namespace stylelint {
 			{ ignore: OneOrMany<'with-var-inside'> }
 		>;
 		'color-hex-alpha': CoreRule<'always' | 'never'>;
-		'color-hex-length': CoreRule<'short' | 'long'>;
+		'color-hex-length': CoreRule<'short' | 'long', {}, AutofixMessage>;
 		'color-named': CoreRule<
 			'never' | 'always-where-possible',
-			{ ignoreProperties: OneOrMany<StringOrRegex>; ignore: OneOrMany<'inside-function'> }
+			{ ignoreProperties: OneOrMany<StringOrRegex>; ignore: OneOrMany<'inside-function'> },
+			{
+				expected: (expected: string, actual: string) => string;
+				rejected: (keyword: string) => string;
+			}
```

Comment:
This illustrates the need of this new argument.
i.e. `color-named` has the arguments in the wrong order